benchmarking/generate_solution.py

```diff
@@ -24,67 +23,6 @@
 logger = logging.getLogger(__name__)
 
 
-@dataclass(frozen=True)
-class ActionsArguments(FlattenedAccess, FrozenSerializable):
```

Comment:
We should keep these here, they are copied from SWE-agent and only specific to the miner. Will make copy/pasting code easier